src/views/PeopleSearch/components/SearchFieldList/components/SearchField/index.tsx

```diff
@@ -18,6 +18,7 @@ interface CommonProps {
   defaultDisabled?: boolean;
   select?: boolean;
   options?: string[] | SelectOption[];
+  defaultLabel: string;
```

Comment:
Since this prop is sometimes not supplied, it needs to be marked as optional:
```suggestion
  defaultLabel?: string;
```